src/modules/AuthDevise.js

```diff
@@ -0,0 +1,145 @@
+import axios from 'axios'
+import store from '../state/store/store'
+import AsyncStorage from '@react-native-community/async-storage'
+
```

Comment:
Is everything in this module in scope for this feature?

Reply:
The token validation process is still very much up in the air due to the oauth situation but it seems wise to have the structure in place at this point